.travis.yml

```diff
@@ -4,3 +4,5 @@ node_js:
 branches:
   only:
     - master
+before_install: npm install -g grunt-cli
```

Comment:
I have to install as a `dependency` or we have to have `npm install --dev` here. That's why the other grunt items are not `devDependencies` BTW.